components/Button.js

```diff
@@ -0,0 +1,18 @@
+export default function Button({ children, link, buttonstyle }) {
+  const openInNewTab = (url) => {
+    const newWindow = window.open(url, '_blank', 'noopener,noreferrer');
+    if (newWindow) newWindow.opener = null;
+  };
+
+  return (
+    <div className="button__container">
+      <button
+        type="button"
+        className={buttonstyle}
+        onClick={() => openInNewTab({ link })}
```

Comment:
Hi Nelson, maybe the onClick method could be passed through props; because not all buttons open a newPage on a new tab